imports/api/inventories.js

```diff
@@ -10,8 +10,9 @@ Meteor.publish('inventories', function() {
 });
 
 Meteor.methods({
-	'inventories.insert'(name) {
+	'inventories.insert'(name, listId) {
```

Comment:
Hmm, thought I commented on this already but now I can't find it (sorry, I'm a noob when it comes to GitHub). It seems redundant to have the list of items _and_ also have the list id in the item, because what if they get out of sync at some point? Maybe we should just discuss this more tomorrow.

Reply:
Yeah that's something that's gotta be sorted out. I left a huge comment in the ui project about this haha. But I did this as a temporary solution since I couldn't figure out how to do the other way. But since you're merging your UI changes, I'm going to follow what you do there. This will be removed later.